tests/test_vcf_readcount_annotator.py

```diff
@@ -157,3 +148,26 @@ def test_hom_ref_genotype(self):
         vcf_readcount_annotator.main(command)
         self.assertTrue(cmp(os.path.join(self.test_data_dir, 'hom_ref.readcount.vcf'), os.path.join(temp_path.name, 'input.readcount.vcf')))
         temp_path.cleanup()
+    
+    @unittest.mock.patch('sys.stdout', new_callable=io.StringIO)
+    def stdout_contains_string(self, vcf_file, readcount_file, expected_output, sample_list, mock_stdout):
+        command = [
+            os.path.join(self.test_data_dir, vcf_file),
+            os.path.join(self.test_data_dir, readcount_file),
+            'DNA'
+        ]
+        if sample_list:
+            command.extend(sample_list)
+        vcf_readcount_annotator.main(command)
+        return expected_output in mock_stdout.getvalue()
+
+    '''
+    def test_duplicate_bam_readcount_entries_same_depth(self):
```

Comment:
You can find test data for this case in https://github.com/griffithlab/pmbio.org/issues/15.